app/src/Readme.tsx

```diff
@@ -3,9 +3,35 @@ import { Navbar } from './Navbar'
 
 export function Readme({ currentPage }: any) {
   return (
-    <div>
-      <Navbar currentPage="#/readme" />
-      Readme...
+    <div className="p-3">
+      <Navbar currentPage="#/" />
+      <h2>What is it?</h2>
+      <p>Nebula is a tool for easing development of Soroban smart contracts.</p>
+      <h2>Development</h2>
+      <p>
+        Nebula aims to provide the following functionality:
+        <ul>
+          <li>Compile contracts</li>
+          <li>Download compiled contracts</li>
+          <li>Deploy contracts</li>
+          <li>An IDE playground</li>
```

Comment:
Are all of the above elements included in Nebula project ?

Reply:
`An IDE playground` will be Pulsar true, I will remove this.

Reply:
Removing one as you requested stops making sense. I am gonna remove all, and let you create a list. I am not sure what to write here.